code/modules/antagonists/melting/melting.dm

```diff
@@ -0,0 +1,120 @@
+
+
+/mob/living/simple_animal/hostile/melting
+	name = "melting"
+	real_name = "melting"
+	desc = "While seemingly nice in appearance, this creature is completely evil."
+	icon = 'icons/mob/melting.dmi'
+	icon_state = "melting_base"
+	icon_living = "melting_base"
+	speed = 2
+	a_intent = INTENT_HARM
+	stop_automated_movement = 1
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	maxHealth = 150
+	health = 150
+	healable = 0
```

Comment:
why is it unhealable?

Reply:
you should be asking why this isn't TRUE FALSE but the real answer is to prevent champions from brute packing the minions to heal them. The slime mobs will be able to heal from going over the slime vomit piles